doc/api/addons.markdown

```diff
@@ -698,3 +698,57 @@ Test it with:
     var result = addon.add(obj1, obj2);
 
     console.log(result); // 30
+
+### AtExit hooks
+#### void AtExit(callback, args)
+
+* `callback`: `void (*)(void*)` - A pointer to the function to call at exit.
+* `args`: `void*` - A pointer to pass to the callback at exit.
+
+Registers exit hooks that run after the event loop has ended, but before the VM is killed.
+
+Callbacks are run in reverse order of registration, i.e. newest first. AtExit takes callback and its arguments as arguments.
```

Comment:
"Callbacks are run in last-in, first-out order. `AtExit` takes two parameters: a pointer to a callback function to run at exit, and a pointer to untyped context data to be passed to that callback."